utils/getScenarioContent.ts

```diff
@@ -159,19 +163,32 @@ export function identifyPendingDeterminationScenario(
 }
 
 /**
- * Identifies the Program Types that count as a federal extension
+ * Identifies the Program Types that count as a Pandemic extension
  **/
-export function isFederalExtension(programType: string): boolean {
+export function isPandemicExtension(programType: string): boolean {
+  const byeValidPandemicExtensions = {
+    PEUC: 'PEUC - Tier 1 Extension',
+    PEUX: 'PEUX - Tier 2 Extension',
+    PEUY: 'PEUY - Tier 2 Augmentation',
+  }
+
+  if (Object.values(byeValidPandemicExtensions).includes(programType)) {
+    return true
+  }
+
+  return false
+}
+
+/**
+ * Identifies the Program Types that count as another extension
+ **/
+export function isOtherExtension(programType: string): boolean {
```

Comment:
Given that we've pulled out all the distinct types until we only have the EU[CXYWZ] family left - do these have a proper category name instead of other?

Reply:
`Regular`? `Normal`? Should we ask EDD?

Reply:
We decided to use EDD's nomenclature and call these `Old Extensions`.